community/people/sh/index.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: person
+title: ShaiAnthony Huff
+image: IMG-0456.jpg
+role: Nuclear Engineering Undergrad
+email: 
+  - huff.shaianthony8@gmail.com 
+room: virtual 
```

Comment:
This all looks great now when I test it.  Given the way the room shows up, you may just want to leave this entry out of your description entirely.

Reply:
okay, I have removed it.